lib/centralisation/ui/pages/Main.dart

```diff
@@ -34,6 +33,32 @@ class LinksScreen extends HookConsumerWidget {
     final section = ref.watch(sectionProvider);
     final favorites = ref.watch(favoritesProvider);
 
+    void showLinkDetails(BuildContext context, Module module) {
```

Comment:
Have you created a file named functions.dart in the tools folder ?
If you have done so, you can move this function there and import it here